guides/v2.1/cloud/howtos/install-components.md

```diff
@@ -161,7 +161,7 @@ You can use CLI commands or directly edit `app/etc/config.php` to enable or disa
 
 ### Modify configurations {#configure}
 
-For projects using {{site.data.var.ece}} **before 2.1.4**, to change settings for your extensions and modules, you should make those changes in all environments as needed. We recommend using similar or matching settings between Staging and Production to fully test functionality. If you have an extension or module using sandbox credentials and settings, you make sure to switch those to live settings if in Production.
+For projects using {{site.data.var.ece}} **before 2.1.4**, to change settings for your extensions and modules, you should make those changes in all environments as needed. We recommend using similar or matching settings between Staging and Production to fully test functionality. If you have an extension or module using sandbox credentials and settings, you must switch those to live settings if in Production.
```

Comment:
This sentence is difficult to understand.  Also-- just a note that "should" is another thing to scrub for

Reply:
I agree.